src/app/app-config.service.ts

```diff
@@ -0,0 +1,31 @@
+import {HttpClient} from "@angular/common/http";
+import {map} from "rxjs";
+import {Injectable} from "@angular/core";
+
+export interface AppConfig {
+  version: string;
+  issuer: string;
+  clientId: string;
+}
+
+@Injectable({
+  providedIn: "root"
+})
+export class AppConfigService {
+  private readonly CONFIGURATION_URL = "config.json";
+  private appConfig?: AppConfig;
+
+  constructor(private httpClient: HttpClient) {
+  }
+
+  public loadConfig() {
```

Comment:
```suggestion
  loadConfig() {
```

how about eslint rule that requires return types?

Reply:
Done :)